controllers/mailing.js

```diff
@@ -21,8 +21,11 @@ const _mailingState = {
 	sent: "sent"
 
 };
+exports.mailingState = _mailingState;
+
+const testMailingSaveText = "^**This is your email notification test / Ceci est votre test de notification par courriel**\n\n";
+const mailingSaveText = "^**Your approval is required for this notification email / Votre approbation est requise pour ce courriel de notification**\n\n";

```

Comment:
It's hard to understand what is the content based on the variable name.

What about something where the name is structured like: <Purpose>-<State>

Here some suggestion:
* emailNoticeDraft or emailNoticeSaveAndTest
* emailNoticeForApproval